.github/workflows/ci.yml

```diff
@@ -79,6 +79,44 @@ jobs:
         run: |
           ./vendor/bin/codacycoverage clover build/coverage/xml
 
+  phpcs:
+    name: PHP_CodeSniffer
```

Comment:
I would prefer we only have one action for static analysis and run all checks there.
I realize the output for failures is arguably nicer but there’s just an uncomfortable amount of repetition between `phpcs` and `phpstan`.
It would probably also mean that if both checks fail, you would only see one of the errors. I think we can live with that.
Linting can be kept separate since we want to run it against multiple versions of PHP.

Reply:
I'll refactor this to use a matrix.

Reply:
Ah, yes, matrix would also work. Thanks.

Reply:
Done.